packages/hardhat/hardhat.config.ts

```diff
@@ -131,11 +123,19 @@ const config: HardhatUserConfig = {
       accounts: [deployerPrivateKey],
     },
   },
+  // configuration for harhdat-verify plugin
+  etherscan: {
+    apiKey: `${etherscanApiKey}`,
+  },
+  // configuration for etherscan-verify from hardhat-deploy plugin
   verify: {
     etherscan: {
       apiKey: `${etherscanApiKey}`,
     },
   },
+  sourcify: {
+    enabled: false,
```

Comment:
its [disabled by default](https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-verify#usage), do we need to write it explicitly?

Reply:
Ohh forgot to mention about it, actually if you don't explicitly mentions it hardhat give you this info message while verifying through `harhdat-verify`: 
![Screenshot 2024-01-06 at 1 06 37 AM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/2000cf31-4b69-46e4-a843-37997e3047f0)

Reply:
got it, thanks!